bokeh/core/properties.py

```diff
@@ -349,7 +352,10 @@ def serializable_value(self, obj):
         value to appear simpler for developer convenience.
 
         """
-        return self.__get__(obj)
+        value = self.__get__(obj)
+        if hasattr(self.descriptor, 'serialize_value'):
```

Comment:
why not have a no-op `serialize_value` in the descriptor base class to avoid this hasattr?